test/remote/gateways/remote_optimal_payment_test.rb

```diff
@@ -143,4 +143,15 @@ def test_invalid_login
     assert_failure response
     assert_equal 'invalid merchant account', response.message
   end
+
+  def test_transcript_scrubbing
+    transcript = capture_transcript(@gateway) do
+      @gateway.purchase(@amount, @credit_card, @options)
+    end
+    transcript = @gateway.scrub(transcript)
+
+    assert_scrubbed(@credit_card.number, transcript)
+    assert_scrubbed(@credit_card.verification_value, transcript)
+    assert_scrubbed('%3CstorePwd%3Etest%3C/storePwd%3E', transcript)
```

Comment:
If you don't want to add a comment here about why you're doing it this way, it might be worth explicitly `CGI.encode`ing it in-place (or equivalent) to make more obvious what you're doing. Just an idea, though.